### ISSUE TITLE
Extensions viewlet: keyboard focus is going on non actionable control.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode#77181]
**Environment Details:**
VSCode Version : 1.36.0(user setup)
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 2.1.1

**Repro Steps:**
1..Open Visual studio code editor.
2.Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3.Using Tab key navigate to the "More" option and select different options( "Show installed Extension, Show outdated extension, Show popular Extension,Etc")
4.Click on any installed extension
5.Try to read all the details about specific extension
6.Navigate to more option button using tab key. 

**Actual:**
Keyboard focus is going on non actionable item.
It is going on extension logo.

**Expected:**
Keyboard focus should not go on non actionable item.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Keyboard only user can get confused about the control.

**MAS Reference:**
MAS 2.1.1 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachment for Reference:**
[VSCode_ Customizations_andAddingExtensions _Keyboard focus is going on non actionable control..zip](https://github.com/microsoft/vscode/files/3380649/VSCode_.Customizations_andAddingExtensions._Keyboard.focus.is.going.on.non.actionable.control.zip)

**Comment by isidorn** [comment]
I agree with this issue. In the detailed view of an extension webview there are some items which are not actionable. We should change that to make them all actionable or remove them from the tabindex.
@sandy081 maybe you provide somce code pointers so we add a help wanted label. What do you think?

**Comment by sandy081** [comment]
Yeah makes sense to me. Here is the place where we render the extension in extension viewlet - 
https://github.com/microsoft/vscode/blob/master/src/vs/workbench/contrib/extensions/browser/extensionsList.ts#L144

**Comment by georgebatalinski** [comment]
@isidorn Can you advise me on what should be actionable here? Should we instead make the reading of the active tab work?

[Extensions View](https://i.postimg.cc/d09XnvdQ/Screen-Shot-2019-08-21-at-9-50-49-PM.png)

1. Via ScreenReader - if we press enter - we get to the details - the text inside the details tab does not read (however, I can navigate using Command + Shift + Tab) 

2. To navigate back to the "Read More" button - (Command + Shift + Tab) - works fine

**Comment by isidorn** [comment]
@georgebatalinski thanks for looking into this and yes you are correct everything is working fine.
What I was seeing before is that there was a license and repository link which were not actionable (however that works just fine for me now).

Thus closing this since it works just fine.
Keyboard forcus is not going to the extension logo.
